JenkinsFile

```diff
@@ -20,7 +20,10 @@ pipeline {
             steps {
                 sh """
                     npm run lint
+                    node build_release_package_json.js
                     npm run test
+                    node build_release_package_json.js
```

Comment:
Why do we have to build release package again?

Reply:
We use the distribution to be deployed for the integration tests. The api request handler gets the SDK version for the user agent string from package.json. If this file doesn't exist it errors